swifttool/client.py

```diff
@@ -20,12 +20,12 @@
 import argparse
 import logging
 import os
-import subprocess
 import sys
 import yaml
 
-from fabric.api import env, execute, hide, parallel, put, sudo
-from swifttool.ring_defintion import SwiftRingsDefinition
+from fabric.api import env, execute, hide, parallel, sudo
+from swifttool.ring_defintion import ringsdef_helper
+from swifttool.manager import capman_helper
```

Comment:
PEP8 imports start with standard lib, then third party tools, then local app imports, starting with `import`s first, then `from foo import bar` imports.  Each group should separated by a blank line.

https://www.python.org/dev/peps/pep-0008/#imports